pegasus/migrations/105_add_contacts_rollup_table.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  change do
+    create_table(:contacts_rollup, charset: 'utf8') do
```

Comment:
nit: Naming convention would suggest the table name should be contact_rollups, based on (e.g.) user_scripts, app_properties, and district_partner_infos.